### DISCUSSION TITLE
Cannot install InfluxDB 1

### DISCUSSION THREAD

**Discussion by redradoa** [tteck/Proxmox#3329] [Help Needed] [answered]
Good morning all
I cannot find a way to install Influxdb with the script. I have shutdown Proxmox and booted it up again. Then I go to the shell and copy the command that we have on the repository:

  bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/influxdb.sh)"

This is what I have on the node shell
Linux proxalbert 6.8.8-2-pve #1 SMP PREEMPT_DYNAMIC PMX 6.8.8-2 (2024-06-24T09:00Z) x86_64

The programs included with the Debian GNU/Linux system are free software;
the exact distribution terms for each program are described in the
individual files in /usr/share/doc/*/copyright.

Debian GNU/Linux comes with ABSOLUTELY NO WARRANTY, to the extent
permitted by applicable law.
Last login: Thu Jul  4 10:50:54 CEST 2024 on pts/0
root@proxalbert:~#   bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/influxdb.sh)"

Down on the task bar I see the status with a spinning wheel
<img width="1682" alt="Screenshot 2024-07-04 at 10 55 41" src="https://github.com/tteck/Proxmox/assets/167394414/26f22eda-5b31-44bd-a155-1650e6db9b5e">

When I click on the task, I see in output a client connection: 127.0.0.1:34170 
Status appears as running, but it stays there for >15m.
Yesterday after many attempts I installed Influx db 2, restarting the machine, etc. 
I am a newby and I would really appreciate your guidance.
Thank you very much in advance
Albert

**Comment by tteck** [comment]
If you paste the command in the shell and hit enter, what happens?

**Reply by redradoa** [reply]
Thank you for answering this fast. I can see that a task is triggered
<img width="800" alt="Screenshot 2024-07-04 at 11 09 13" src="https://github.com/tteck/Proxmox/assets/167394414/c7c17675-26dd-4ded-9335-04a5c57b1697">
<img width="1655" alt="Screenshot 2024-07-04 at 11 09 44" src="https://github.com/tteck/Proxmox/assets/167394414/1cd94a4c-de94-4312-a551-ebdb20d4327f">

But I don't get the LXCcreation dialog

**Reply by tteck** [reply, marked as answer]
🌐 It seems that you're facing a network problem, _probably_ linked to DNS.

**Reply by redradoa** [reply]
Is there anywhere where I can look into?

**Reply by redradoa** [reply]
I have checked the node DNS settings and they match the ones that I have on my router.

**Reply by tteck** [reply]
What do you have for your DNS server?

![Screenshot 2024-07-04 5 15 15 AM](https://github.com/tteck/Proxmox/assets/21342670/117ccdb6-fb42-4235-8f11-bc0c4552260a)

**Reply by redradoa** [reply]
<img width="862" alt="Screenshot 2024-07-04 at 11 18 07" src="https://github.com/tteck/Proxmox/assets/167394414/0617485d-2d26-4b74-93eb-23c921778506">
The same DNS server as my router

**Reply by tteck** [reply]
Change it to your router/gateway IP and try

**Reply by redradoa** [reply]
Done it
<img width="767" alt="Screenshot 2024-07-04 at 11 19 49" src="https://github.com/tteck/Proxmox/assets/167394414/0f811ac0-cd5a-461c-9d03-1f074a133aef">
Rebooted Proxmox
Copied the script on the shell
<img width="795" alt="Screenshot 2024-07-04 at 11 21 52" src="https://github.com/tteck/Proxmox/assets/167394414/0bda2edc-0cc8-48ad-ab89-b026bcae3770">
Task launched, but it keeps spinning

**Reply by tteck** [reply]
The task is the shell
What was the default Search domain?

**Reply by redradoa** [reply]
In the meantime, allow me to buy you a coffee.

**Reply by tteck** [reply]
Try Search domain `lan` and DNS server `8.8.8.8`

**Reply by redradoa** [reply]
It has worked!
<img width="238" alt="Screenshot 2024-07-04 at 11 31 06" src="https://github.com/tteck/Proxmox/assets/167394414/c0430983-9b73-4d6c-99ae-4a2d57f85db7">
Should I leave this settings as they are now permanently?
Thank you for your support and for your great work!!

**Reply by tteck** [reply]
Try setting your DNS server back to the router IP and see if it still works

**Reply by redradoa** [reply]
I have run the script with the default settings but I have not been asked for influxdb version.
This is the summary:
    ____      ______           ____  ____ 
   /  _/___  / __/ /_  ___  __/ __ \/ __ )
   / // __ \/ /_/ / / / / |/_/ / / / __  |
 _/ // / / / __/ / /_/ />  </ /_/ / /_/ / 
/___/_/ /_/_/ /_/\__,_/_/|_/_____/_____/  
 
Using Default Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: Automatic Login
Using Container ID: 101
Using Hostname: influxdb
Using Disk Size: 8GB
Allocated Cores 2
Allocated Ram 2048
Using Bridge: vmbr0
Using Static IP Address: dhcp
Using Gateway IP Address: Default
Using Apt-Cacher IP Address: Default
Disable IPv6: No
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server Address: Host
Using MAC Address: Default
Using VLAN Tag: Default
Enable Root SSH Access: No
Enable Verbose Mode: No
Creating a InfluxDB LXC using the above default settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 101 was successfully created.
 ✓ Started LXC Container
 ✓ Completed Successfully!

root@proxalbert:~# 

I cannot access influxdb
Any clue?

**Reply by tteck** [reply]
SSH or Proxmox shell?

**Reply by tteck** [reply]
🏁 The script runs without any problems from this end.

```
    ____      ______           ____  ____ 
   /  _/___  / __/ /_  ___  __/ __ \/ __ )
   / // __ \/ /_/ / / / / |/_/ / / / __  |
 _/ // / / / __/ / /_/ />  </ /_/ / /_/ / 
/___/_/ /_/_/ /_/\__,_/_/|_/_____/_____/  
 
Using Advanced Settings
Using Distribution: debian
Using debian Version: 12
Using Container Type: 1
Using Root Password: Automatic Login
Container ID: 112
Using Hostname: influxdb
Using Disk Size: 8
Allocated Cores: 2
Allocated RAM: 2048
Using Bridge: vmbr0
Using IP Address: dhcp
Using Gateway IP Address: Default
Using APT-Cacher IP Address: Default
Disable IPv6: yes
Using Interface MTU Size: Default
Using DNS Search Domain: Host
Using DNS Server IP Address: Host
Using MAC Address: AE:1A:60:07:B7:59
Using Vlan: Default
Enable Root SSH Access: no
Enable Verbose Mode: no
Creating a InfluxDB LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using thin-pool for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 112 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.250 
 ✓ IPv4 Internet Connected
 ✗ IPv6 Internet Not Connected
 ✓ DNS Resolved github.com to 140.82.114.3
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Set up InfluxDB Repository
Which version of InfluxDB to install? (1 or 2) 2
 ✓ Installed InfluxDB
Would you like to add Telegraf? <y/N> 
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!
```

**Reply by tteck** [reply]
![Screenshot 2024-07-04 5 44 40 AM](https://github.com/tteck/Proxmox/assets/21342670/ca2e0892-4d48-45e6-ad1b-257a736d6b26)

**Reply by redradoa** [reply]
> Try setting your DNS server back to the router IP and see if it still works

When I revert to the router IP it doesn't work anymore. I have to put 8.8.8.8 as advised

**Reply by tteck** [reply]
I would add another DNS server with `8.8.4.4`

**Reply by redradoa** [reply]
Again, thank you for your great support which I really appreciate.
I have created a new LXC with the default values and it does not ask me which version of influxdb I want. 
When I do it with the advanced accepting all the default values but the one about root ssh access that I change to yes, it does  give me the option to select influxdb 1 or 2.
<img width="959" alt="Screenshot 2024-07-04 at 11 55 41" src="https://github.com/tteck/Proxmox/assets/167394414/49c56978-f34c-497f-8128-60d8cc5ca4b6">
Now I can access influxdb in its ip:8888 without issues too.
<img width="596" alt="Screenshot 2024-07-04 at 12 00 41" src="https://github.com/tteck/Proxmox/assets/167394414/c1026374-4f53-46f5-b130-6e12da8acede">
If you need any other clarification from myself, let me know. I will also add another DNS server as you have specified
Thanks
Albert

**Reply by tteck** [reply]
Sounds like a browser cache issue. If you delete your browser cache, default settings will ask like Advanced settings.
